### PR TITLE
Added Geoff's introduction to the home page

### PR DESCRIPTION
**Pull request by rosscastle**
Added a large amount of text to the home. 

Text written by Geoff Vallis and tweaked by me. As agreed with GV I added some plots to brighten up the page.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,10 +1,118 @@
...
+   Figure 2. Simulations for the Trappist-1 Habitable Atmosphere Intercomparison (THAI) project. Trappist 1-E is a tidally-locked rocky exoplanet and is a promising candidate for habitable surface conditions. Figure produced by Dr Matthew Henry.
+
+The Future
+^^^^^^^^^^
+Isca, and this documentation, is constantly under development. A simple cloud scheme has recently been incorporated into Isca, and models of other planetary bodies are continually being developed. The next major model addition will be a fully interactive ocean as an integrated component of Isca and therefore configurable with the Python interface and flexible in its nature and use.  We invite contributions from users to add to its capabilities and improve its documentation. 
+
+.. figure:: index_fig_qbo.png
+   :scale: 50
+
+   Figure 3. The Quasi-Biennial Oscillation (QBO), an amazing wave driven periodic reversal of the stratospheric tropical winds on Earth. This was produced in Isca using the convective gravity wave parameterisation modified by Prof. Chaim Garkfinel. Figure produced by Ross Castle.
```
Comment (dennissergeev) [inline docs/source/index.rst:87]: > an amazing wave driven periodic reversal 
I understand your excitement, but shouldn't we keep the docs neutral? :)

File: docs/source/index.rst [hunk 2]
```diff
@@ -1,10 +1,118 @@
...
+* Cases with realistic, Earth-like continents
+
+Planets
+^^^^^^^
+Earth is, naturally, the planet to which Isca has most often been applied. However, models of varying degrees of sophistication of Mars, Jupiter, Titan, Trappist 1-e, and a generic tidally-locked exoplanet have or are being constructed. 
+
+.. figure:: index_fig_trappist.png
+   :scale: 75
+
+   Figure 2. Simulations for the Trappist-1 Habitable Atmosphere Intercomparison (THAI) project. Trappist 1-E is a tidally-locked rocky exoplanet and is a promising candidate for habitable surface conditions. Figure produced by Dr Matthew Henry.
```
Comment (dennissergeev) [inline docs/source/index.rst:78]: TRAPPIST-1 and TRAPPIST-1e

File: docs/source/index.rst [hunk 3]
```diff
@@ -1,10 +1,118 @@
...
+* The Held-Suarez test case
+* An aquaplanet with a slab ocean and grey radiation
+* A MiMA-like test case using RRTM
+* An axi-symmetric model
+* A giant planet (Jupiter-like)
+* Cases with realistic, Earth-like continents
+
+Planets
+^^^^^^^
+Earth is, naturally, the planet to which Isca has most often been applied. However, models of varying degrees of sophistication of Mars, Jupiter, Titan, Trappist 1-e, and a generic tidally-locked exoplanet have or are being constructed.
```
Comment (dennissergeev) [inline docs/source/index.rst:73]: ~~Trappist 1-e~~

TRAPPIST-1e

File: docs/source/index.rst [hunk 4]
```diff
@@ -1,10 +1,118 @@
...
+
+Options and configurability
+---------------------------
+
+The main code is written in Fortran 90 to which we provide a Python front end. The front end enables many model configurations, parameter changes and output options to be set without going into the code itself, but if you wish to make more substantial changes then editing the Fortran may be necessary. 
+
+Dynamical cores and variants:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Primitive equation, spectral model on the sphere. This may also be configured as a zonally-symmetric model
```
Comment (dennissergeev) [inline docs/source/index.rst:29]: Obviously later it would be logical to add hyperlinks to each of the components

File: docs/source/index.rst [hunk 5]
```diff
@@ -1,10 +1,118 @@
...
+
+Isca's user friendly Python front end allows users to quickly and easily configure their experiments. Test experiments are included to aid the first-time user in understanding the framework, but can also be used as a basis for more specialised experiments. Because of it flexibility and intuitive interface, Isca now has many users thus creating an Isca community that, with the addition of the documentation you can find here, are able to support fellow researchers with ideas and solutions to problems.  The code is open source and downloadable from `github <https://github.com/ExeClim/Isca>`_, and if you have questions please raise an issue there. 
+
+The various options for Isca are described in detail in the links in the left column (where appropriate references are also given), and here we just give an overview of what is available. Please note that, since Isca is constantly being developed and improved, not all of the options are as yet fully documented.  We hope that you, the user, may also wish to contribute to Isca's development and documentation.
```
Comment (dennissergeev) [inline docs/source/index.rst:19]: add
```
(see the `Issues <https://github.com/ExeClim/Isca>`_ page)
```

at the end

File: docs/source/index.rst [hunk 6]
```diff
@@ -1,10 +1,118 @@
...
+Dynamical cores and variants:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Primitive equation, spectral model on the sphere. This may also be configured as a zonally-symmetric model 
+* Shallow water model on the sphere
+* Column model, for example for radiative-convective equilibrium studies
+
+Radiative forcing
+^^^^^^^^^^^^^^^^^
+* Thermal relaxation: Held-Suarez, variations for exoplanets, and a 'top--down' scheme that provides a self-consistent determination of the tropopause
```
Comment (dennissergeev) [inline docs/source/index.rst:35]: variations for **terrestrial** exoplanets

File: docs/source/index.rst [hunk 7]
```diff
@@ -1,10 +1,118 @@
...
+* A simple cloud scheme based on relative humidity and inversion strength. 
+
+Surface properties
+^^^^^^^^^^^^^^^^^^
+* Specified sea-surface temperatures
+* A slab ocean with a prognostic SST
+* A swamp land surface with an evaporative resistance
+* A simple bucket model 
+* A simple vegetation model
+* Realistic Earth continental outlines, or user-configurable idealized continental configurations
```
Comment (dennissergeev) [inline docs/source/index.rst:58]: plus topography?

File: docs/source/index.rst [hunk 8]
```diff
@@ -1,10 +1,118 @@
...
+* Realistic Earth continental outlines, or user-configurable idealized continental configurations 
+
+Test cases
+^^^^^^^^^^
+Various out of the box test cases are available that should run with no additional configuration. These include:
+
+* The Held-Suarez test case
+* An aquaplanet with a slab ocean and grey radiation
+* A MiMA-like test case using RRTM
+* An axi-symmetric model
```
Comment (dennissergeev) [inline docs/source/index.rst:67]: axisymmetric (w/o a hyphen)

File: docs/source/index.rst [hunk 9]
```diff
@@ -1,10 +1,118 @@
...
+* A giant planet (Jupiter-like)
```
Comment (dennissergeev) [inline docs/source/index.rst:68]: A gas giant planet

File: docs/source/index.rst (hunk 2 above)
Comment (dennissergeev) [reply docs/source/index.rst:78]: Also, add a link to THAI please
http://www.nexss.info/community/workshops/thai

File: docs/source/index.rst [hunk 10]
```diff
@@ -1,10 +1,118 @@
...
+The Future
+^^^^^^^^^^
+Isca, and this documentation, is constantly under development. A simple cloud scheme has recently been incorporated into Isca, and models of other planetary bodies are continually being developed. The next major model addition will be a fully interactive ocean as an integrated component of Isca and therefore configurable with the Python interface and flexible in its nature and use.  We invite contributions from users to add to its capabilities and improve its documentation. 
+
+.. figure:: index_fig_qbo.png
+   :scale: 50
+
+   Figure 3. The Quasi-Biennial Oscillation (QBO), an amazing wave driven periodic reversal of the stratospheric tropical winds on Earth. This was produced in Isca using the convective gravity wave parameterisation modified by Prof. Chaim Garkfinel. Figure produced by Ross Castle.
+
+References
```
Comment (dennissergeev) [inline docs/source/index.rst:89]: All these references should be moved to the Reference page (with citations keys as appropriate)

**Review by dennissergeev — CHANGES_REQUESTED** [review]
Nice work @gkvallis and @rosscastle !

There are a few minor changes to make, and moving the references to the page with all common bibliography

File: docs/source/index.rst [hunk 11]
```diff
@@ -1,10 +1,118 @@
...
+Options and configurability
+---------------------------
+
+The main code is written in Fortran 90 to which we provide a Python front end. The front end enables many model configurations, parameter changes and output options to be set without going into the code itself, but if you wish to make more substantial changes then editing the Fortran may be necessary. 
+
+Dynamical cores and variants:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Primitive equation, spectral model on the sphere. This may also be configured as a zonally-symmetric model 
+* Shallow water model on the sphere
```
Comment (sit23) [inline docs/source/index.rst:30]: Barotropic vorticity equation

**Comment by sit23** [comment]
Looks good to me!

**Comment by sit23** [comment]
Perhaps this isn't the right place for it, but would it be worth mentioning somewhere about how to cite Isca if you use it? 

E.g. v1 Isca paper for most things
The gravity paper that GV and I wrote if using Socrates etc?

**Comment by rosscastle** [comment]
Hi Both,

Thanks for these, agree with all. I will go back and add links to pages when more docs are written.

The references section should perhaps be called 'Publications', as it was supposed to be a list of publications created by Isca, not citations which is why I didn't add to the file. If I change to publications would this be ok?

RE citing, I can certainly add in more papers and make it more explicit. How about:


Please reference Isca in any work. For most applications please reference:
V1 Paper ref

If your work uses SOCRATES then please reference:
Gravity paper ref

If your work uses the Mars, Jupiter, S/W models, please reference
Hierarchy paper ref

Other papers that may relevant if you use particular aspects of Isca are:...

What is S/W @sit23 ?

**Comment by sit23** [comment]
Ah - sorry Ross. S/W = shallow water

**Comment by gkvallis** [comment]
I’m missing something here. In the version I sent Ross there was exactly information about how to cite papers.

**Comment by sit23** [comment]
Yes - you're right. Sorry - I assumed the papers listed at the end was just a list, and missed that you'd labeled them individually. I'd appreciate if the gravity paper could go in as the Socrates reference. But I wonder if this section could be labeled 'How to cite Isca' or something like that, rather than just 'references' to avoid the confusion I just had?

**Comment by rosscastle** [comment]
> I’m missing something here. In the version I sent Ross there was exactly information about how to cite papers.
> […](#)

Your citation information is still there, the question whether we should ask them to cite other papers if they use specific bits, e.g. if using Socrates then they should cite the gravity wave paper instead of the original Isca paper. What do you think? Should we ask them to cite more than one?

**Comment by sit23** [comment]
@rosscastle I think they should be asked to cite more than one if they use extra features, at least until we release a 'version 2' paper that includes all of the updates we've made since version 1.

**Comment by rosscastle** [comment]
I think a cite section is a good idea. I'll make those changes :)

**Comment by gkvallis** [comment]
Yes, change 'references' to 'how to cite Isca' or similar.    Indeed there is no reason why two papers should not be cited, the original GMD one plus an additional as needed, depending on context.

**Comment by rosscastle** [comment]
Hi all, 

Changes made, let me know if this is OK to merge now?

Note: The figures are much smaller then they appear when using the 'view file' option on github.

**Review by dennissergeev — APPROVED** [review]
LGTM

**Review by sit23 — APPROVED** [review]
Looks good to me - thanks @rosscastle @gkvallis

**Pull request merged**